bin/run_schema_transform.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash 
```

Comment:
Maybe move this to a container directory, with the Dockerfile, and then leave a new `run_schema_transform.sh` that wraps the docker or singularity run as it would be used.

Reply:
To clarify, what I would want here is something that looks like this:

```
[ -z ${PROCESSED_JSON}+x ] && echo "Error: variable PROCESSED_JSON is not defined." && exit 1 
[ -z ${INPUT_JSON}+x ] && echo "Error: variable INPUT_JSON is not defined." && exit 1 
[ -z ${SCHEMA_TRANSFORM}+x ] && echo "Error: variable SCHEMA_TRANSFORM is not defined." && exit 1 

if [ "$container" == "docker ]; then
    docker run -v $INPUT_JSON:/data/input.json -v $SCHEMA_TRANSFORM:/data/schema_transform.jslt -v $PROCESSED_JSON:/data/result.txt -e <any-var-neeed-by-run_schema_transform> <container-image> /bin/run_schema_transform_container.sh
else
    singularity run ....
fi
```

that in turn calls the logic that was on this script in the container. This should be the entry point for usage, don't expect the external process calling this to know the docker/singularity logic for running it. And then call the above container from bats, setting in the environment if you are using docker or singularity.